lisp/kbd-mode.el

```diff
@@ -150,32 +150,42 @@ If SHOW-MACROS is nil, don't highlight macros of the form
         (font-lock-add-keywords 'kbd-mode macro-regexp)
       (font-lock-remove-keywords 'kbd-mode macro-regexp))))
 
+;; How many times have it been done?
+(defun backward-symbol ()
+  "Like `forward-symbol' but backwards."
+  (forward-symbol -1))
+
```

Comment:
`forward-symbol -arg` is perhaps not the best idiom, but for one single use I wouldn't define a new function for this—especially not with a non-qualified name

Reply:
Sure!
I just don't know how to do it differently ;)
Will try to change